worlds/kh1/Client.py

```diff
@@ -77,6 +77,20 @@ def _cmd_advanced_logic(self):
             self.output(str(self.ctx.slot_data["advanced_logic"]))
         else:
             self.output("Unknown")
+    
+    def _cmd_required_postcards(self):
+        """Prints the number of postcards required if goal is set to postcards"""
+        if "required_postcards" in self.ctx.slot_data.keys():
+            self.output(str(self.ctx.slot_data["required_postcards"]))
+        else:
+            self.output("Unknown")
```

Comment:
If you expect this else to ever be reached, I would recommend having a more wordy message. Maybe something like "Goal is not set to postcards"?

Reply:
I actually have this "else" here if the player has not connected yet, or is playing an old seed from before this option was made available.  Maybe I should change "Unknown" to "Not Available" or something like that?

Reply:
Maybe something like that, yeah o: Less chance of them putting up a bug report for it